x/ibc/03-connection/keeper/keeper.go

```diff
@@ -20,14 +20,16 @@ import (
 // Keeper defines the IBC connection keeper
 type Keeper struct {
 	storeKey     sdk.StoreKey
-	cdc          *codec.Codec
+	aminoCdc     *codec.Codec    // amino codec. TODO: remove after clients have been migrated to proto
```

Comment:
Why do you still need the amino codec? The hybrid codec will use Amino for everything JSON (i.e. client).